core/src/main/java/fr/sncf/osrd/railjson/schema/schedule/RJSAllowance.java

```diff
@@ -16,6 +16,8 @@
         public double beginPosition = Double.NaN;
         @Json(name = "end_position")
         public double endPosition = Double.NaN;
+        @Json(name = "capacity_speed_limit")
+        public double capacitySpeedLimit = -1;
```

Comment:
Why is this set to -1 here ? And not 0 or 30 / 3.6 ?

Reply:
To leave it at the default value set in the parser. I wanted to put NaN but it caused issues when serializing things.